src/DittoMachine.sol

```diff
@@ -275,7 +332,7 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver {
             );
             uint256 subsidyDiv2 = subsidy >> 1;
             // half of fee goes into subsidy pool, half to previous clone owner
-            cloneIdToSubsidy[cloneId] += subsidyDiv2;
+            protoIdToSubsidy[protoId] += subsidy >> 1;
```

Comment:
replace with `subsidyDiv2`.

Reply:
I don't know why or how this was changed but i put it back to normal in the last commit